### PR TITLE
Not showing disabled build results on default

### PR DESCRIPTION
**Pull request by gsouzab**
If you haven't done so already, please read the CONTRIBUTING.md file to learn how we work and what we expect from all contributors.

https://github.com/openSUSE/open-build-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: src/api/app/models/local_build_result/for_package.rb [hunk 1]
```diff
@@ -33,8 +33,8 @@ def local_build_result(result, status)
                            code: status['code'], state: result['state'], details: status['details'])
     end
 
-    def excluded?(status)
-      !show_all && status == 'excluded'
+    def excluded_or_disabled?(status)
+      !show_all && (status == 'excluded' || status == 'disabled')
```
Comment (hennevogel) [inline src/api/app/models/local_build_result/for_package.rb]: This is getting complicated. Do you have another idea how to do this?

File: src/api/app/models/local_build_result/for_package.rb [hunk 2]
```diff
@@ -33,8 +33,12 @@ def local_build_result(result, status)
...
     end
 
-    def excluded?(status)
-      !show_all && status == 'excluded'
+    def excluded_or_disabled?(status)
+      if show_all
+	return false
+      end
+	 
+      status == 'excluded' || status == 'disabled'
```
Comment (Ana06) [inline src/api/app/models/local_build_result/for_package.rb]: what about `status.in?['excluded', 'disabled']`? :wink:

File: src/api/app/models/local_build_result/for_package.rb [hunk 3]
```diff
@@ -33,8 +33,12 @@ def local_build_result(result, status)
                            code: status['code'], state: result['state'], details: status['details'])
     end
 
-    def excluded?(status)
-      !show_all && status == 'excluded'
+    def excluded_or_disabled?(status)
+      if show_all
```
Comment (Ana06) [inline src/api/app/models/local_build_result/for_package.rb]: it is nice to write short conditions in one line: `return false if show_all`

File: src/api/app/models/local_build_result/for_package.rb (hunk 2 above)
Comment (gsouzab) [reply src/api/app/models/local_build_result/for_package.rb]: Went with `['excluded', 'disabled'].include?(status)`

**Comment by hennevogel** [comment]
@gsouzab now if you squash the two commits, we are done! Hooray :tada::balloon:

**Comment by hennevogel** [comment]
You can also see proof that it works, here: 
https://obs-reviewlab.opensuse.org/gsouzab-disabled_build_results/package/show/home:Admin/ruby

**Comment by gsouzab** [comment]
Hi @Ana06 and @hennevogel, I was struggling a bit with the squash but I think I got it.

**Comment by Ana06** [comment]
@gsouzab will you squash them? If not I will do it for you so that we can merge this :wink:

**Comment by gsouzab** [comment]
Hi @Ana06, I agree with you. Makes sense to have only one commit in this case.
I'll squash them up.

I also want to change the following messages from **excluded** to **excluded/disabled**:
captura de tela de 2018-12-18 12-57-35

Do you agree?

**Comment by Ana06** [comment]
@gsouzab 

> I also want to change the following messages from excluded to excluded/disabled:
> Do you agree?

Good idea :wink:

**Pull request merged**